build.gradle

```diff
@@ -27,6 +27,10 @@ reckon {
 
 repositories {
     mavenLocal()
+    maven {
+        name = 'modrinth'
+	url = "https://api.modrinth.com/maven"
+	}
```

Comment:
Did you forget the other line? 🤔

Reply:
No I did not forget I just did not see it